src/main/java/com/connectCo/domain/coupon/coupon.java

```diff
@@ -0,0 +1,46 @@
+package com.connectCo.domain.coupon;
+
+import com.connectCo.domain.BaseEntity.baseEntity;
+import com.connectCo.domain.enums.CouponType;
+import com.connectCo.domain.store.store;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Date;
+import java.util.UUID;
+
+@Getter
+@Setter
+@Entity
+@NoArgsConstructor
+public class coupon extends baseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "coupon_id", nullable = false)
+    private UUID id;
+
+    @ManyToOne
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
```
fetch 타입으로 LAZY를 주는 게 SQL 문 덜 써서 웬만하면 무조건 달아주는 걸 추천해